imports/ui/components/newTask/newTask.html

```diff
@@ -1,12 +1,64 @@
 <template name="newTask">
+  <h2>New Task</h2>
     <form class="new-task">
+      <div class="form-group">
       <input type="text" name="text" placeholder="Task Description" />
-      <select id="category-select">
-        <option disabled="disabled" selected="selected">Category</option> 
-        {{#each categories}}
-          <option value="{{this}}" name="category">{{this}}</option>
-        {{/each}}
-      </select>
-      <!--<input type="text" name="due" class="form-control" id="duedate" placeholder="Due Date">-->
+      </div>
+
+        {{> datepicker}}
+
+        <button name="setPriority" class="glyphicon glyphicon-th-large setPriority" type="button" data-container="body" data-trigger="focus" data-toggle="popover" data-placement="bottom"></button>
+        <div id="popover-content" class="hide">
+        {{> choosePriority}}
+        </div>
+        
     </form>
+</template>
+
+<template name="choosePriority">
```

Comment:
Ideally it's 1 template per file. We can keep this, and if it becomes annoying later we'll fix it.